pombola/core/views.py

```diff
@@ -128,6 +144,12 @@ def get_context_data(self, **kwargs):
         context = super(BasePlaceDetailView, self).get_context_data(**kwargs)
         context['place_type_count'] = models.Place.objects.filter(kind=self.object.kind).count()
         context['related_people'] = self.object.related_people()
+        if settings.ENABLED_FEATURES['projects']:
+            # The number of projects associated with the place is used
+            # in the link text in the object_menu_links:
+            context['projects_sub_link_text'] = 'CDF Projects ({0})'.format(
```

Comment:
I guess as it's only a number at the end, you could translate the "CDF Projects" and concatenate on the number, all in the template. Probably not worth it at the mo!